renderer/consts/actions.ts

```diff
@@ -0,0 +1,33 @@
+export enum AlertActionsTypes {
+  ADD_ASYNC_ALERT = 'ADD_ASYNC_ALERT',
+  REMOVE_ASYNC_ALERT = 'REMOVE_ASYNC_ALERT'
+}
+
+export enum ConsoleActionsTypes {
+  UPDATE_CONSOLE = 'UPDATE_CONSOLE',
+  CLEAR_CONSOLE = 'CLEAR_CONSOLE',
+  TOGGLE_CONSOLE = 'TOGGLE_CONSOLE',
+  TOGGLE_SCROLL = 'TOGGLE_SCROLL'
+}
+
+export enum PeripheralActionsTypes{
+  UPDATE_PERIPHERALS = 'UPDATE_PERIPHERALS',
+  PERIPHERAL_RENAME = 'PERIPHERAL_RENAME'
+}
+
+export enum EditorActionsTypes{
+  EDITOR_UPDATE = 'UPDATE_EDITOR',
```

Comment:
Should this named be `UPDATE_EDITOR` instead of `EDITOR_UPDATE`? I just named it editor_update because that's the function name, but I am not sure.

Reply:
yea, I think `UPDATE_EDITOR` sounds good.